mappings/chicagoISAC.py

```diff
@@ -0,0 +1,42 @@
+from .json import JSONMapping
+
+class ChicagoISACMapping(JSONMapping):
+    def __init__(self, source):
+        super().__init__(source, {})
+        self.mapping = self.createMapping()
+    
+    def createMapping(self):
+        return {
+            'title': ('title', '{0}'),
+            'authors': ('authors', '{0}|||true'),
+            'dates': [('publicationDate', '{0}|publication_date')],
+            'publisher': ('publisher', '{0}'),
+            'identifiers': 
+                [('isbn', '{0}|isbn')]
+            ,
+            'is_part_of': ('series', '{0}|series'),
+            'spatial': ('publisherLocation', '{0}|publissherLocation'),
```

Comment:
small thing I missed, is that a typo? Looks like you have an extra 's' in 'publissherLocation'

Reply:
That is a type. Thanks for pointing it out.